chains/solana/contracts/programs/ccip-router/src/instructions/v1/mod.rs

```diff
@@ -10,10 +10,9 @@ pub mod token_admin_registry;
 // Private modules //
 /////////////////////
 mod config;
-mod fee_quoter;
+mod fees;
```

Comment:
Any particular reason for this rename? Not opposed to it, but it loses a bit of connection with the EVM codebase.

Reply:
Because fee quoter is now a separate program altogether, and this file is doing other things than just invoking it (e.g. wrapping native tokens and actually transferring tokens), I renamed it to prevent confusion. 
The thing that EVM calls "Fee Quoter" is our Fee Quoter program; this submodule of the router is not it. I'm happy to rename it to something else if you have any suggested names 😄